### PR TITLE
Created soumdatta81.json

### PR DESCRIPTION
**Pull request by soumdatta81**

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by accodes21 — APPROVED** [review]
Looks great

File: data/soumdatta81.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "September 2022",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing",
```
Comment (eddiejaoude) [inline data/soumdatta81.json:25]: Trailing comma is invalid

```suggestion
      "description": "Started freelancing"
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Invalid json, comment left inline

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**